Directory.Build.props

```diff
@@ -17,7 +17,10 @@
 		<PackageLicenseExpression>MIT</PackageLicenseExpression>
 		<RepositoryUrl>https://github.com/kmdlogic/kmd-logic-gateway-automation</RepositoryUrl>
 		<RepositoryType>git</RepositoryType>
-		<GenerateAssemblyVersionAttribute>false</GenerateAssemblyVersionAttribute>
+
+		<VersionPrefix Condition=" '$(VersionPrefix)' == '' ">0.1.0</VersionPrefix>
```

Comment:
Verify if `VersionSuffix` is still required in `src/Kmd.Logic.Gateway.Automation.Tool/Kmd.Logic.Gateway.Automation.Tool.csproj` and library as well

Reply:
Removed as agreed on Teams :)